localized/games/wordwindow/wordwindow_level_en.tsv

```diff
@@ -194,11 +194,11 @@ en-US	14	3	2	4	"Namita has 13 flowers in her garden. Gauri has 8 flowers. How ma
-en-US	15	1	1	5	"Ms. Basanti is playing a hide-and-seek game with her 11 students. She finds 3 students. How many students are still hiding?"	Ms. Basanti is playing a hide-and-seek game with her 11 students. She finds 3 students. How many students are still hiding?	3	5	8	11	students	students	students	students	11 = 3 + <8>	3	3	8	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
+en-US	15	1	1	5	"Miss Basanti is playing a hide-and-seek game with her 11 students. She finds 3 students. How many students are still hiding?"	Miss Basanti is playing a hide-and-seek game with her 11 students. She finds 3 students. How many students are still hiding?	3	5	8	11	students	students	students	students	11 = 3 + <8>	3	3	8	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
 en-US	15	1	2	5	"There are 15 students in a classroom. 9 of the students are wearing long pants. The rest are wearing short pants. How many students are wearing short pants?"	There are 15 students in a classroom. 9 of the students are wearing long pants. The rest are wearing short pants. How many students are wearing short pants?	3	6	9	15	students	students	students	students	15 = 9 + <6>	2	9	6	0	0					wordwindow-stone-red.png	wordwindow-stone-yellow.png		
 en-US	15	1	3	5	"Jaswant has 17 balloons. 8 balloons are red and the rest are green. How many balloons are green?"	Jaswant has 17 balloons. 8 balloons are red and the rest are green. How many balloons are green?	7	8	9	10	balloons	balloons	balloons	balloons	17 = 8 + <9>	3	8	9	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
 en-US	15	1	4	5	"Priyanka and Kajal caught 14 fireflies altogether. Priyanka caught 5 fireflies. How many fireflies did Kajal catch?"	Priyanka and Kajal caught 14 fireflies altogether. Priyanka caught 5 fireflies. How many fireflies did Kajal catch?	6	9	12	14	fireflies	fireflies	fireflies	fireflies	14 = 5 + <9>	2	5	9	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
-en-US	15	1	5	5	"Ms. Jagat has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?"	Miss Jagat has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?	5	7	9	12	students	students	students	students	12 = <5> + 7	1	5	7	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
+en-US	15	1	5	5	"Miss Payal has 12 students in her class. Some students are absent and 7 students are present. How many students are absent?"	Miss Payal has 12 students in her class. Some students are absent and@7 students are present. How many students are absent?	5	7	9	12	students	students	students	students	12 = <5> + 7	1	5	7	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
```

Comment:
Remove `@` from the TTS text. Have you checked how the TTS is coming with the `@` symbol?
Remove `@` from all the text of the TSV file

Reply:
Removed @ from the TSV file